airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/job.py

```diff
@@ -109,11 +137,19 @@ def job_get_result(self, response: Optional[requests.Response] = None) -> Option
     def job_update_state(self, response: Optional[requests.Response] = None) -> None:
         if response:
             self.job_state = response.json().get("data", {}).get("node", {}).get("status")
-            self.log_state()
+            if self.job_state == ShopifyBulkStatus.RUNNING.value:
+                self.log_running_job_state()
+            else:
+                self.log_state()
 
     def on_created_job(self, **kwargs) -> None:
         pass
 
+    def on_canceled_job(self, response: requests.Response) -> AirbyteTracedException:
+        raise ShopifyBulkExceptions.BulkJobCanceled(
```

Comment:
We will now raise the config error, when this will happen, once we got more cases like this, we will handle this error as well. For now there is no clear vision of what to do with this status.

Reply:
Why would this be a config_error? In my mind, a config error is an error the user can take action on. What action would the user take here? Can we make it explicit in the error message?

Reply:
Changed to `FailureType.system_error`, as this is not actionable for the Customer, we should be able to retry the canceled job, if this becomes widespread.